cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/oauth/OAuthAccessToken.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2025 Cask Data, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package io.cdap.cdap.datapipeline.oauth;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * OAuth access token, with related metadata required to retrieve a long-lived access token.
+ */
+public class OAuthAccessToken {
+  private final String accessToken;
+  private final String redirectURI;
```

Comment:
Wondering does `accessToken` really needs `redirectUri`? It makes sense to include it in `refreshToken`. Do you have an example how is it used in case of `accessToken`?